sources/packages/web/src/services/EducationProgramOfferingService.ts

```diff
@@ -297,7 +297,7 @@ export class EducationProgramOfferingService {
   async offeringBulkInsert(
     file: Blob,
     validationOnly: boolean,
-    onUploadProgress: (progressEvent: FileUploadProgressEventArgs) => void,
+    onUploadProgress: (progressEvent: AxiosProgressEvent) => void,
```

Comment:
just wanted to double-check, I see that, `bytes` and `loaded` are the 2 mandatory properties in AxiosProgressEvent https://www.jsdocs.io/package/axios#AxiosProgressEvent, and  `bytes` was not in FileUploadProgressEventArgs. So, I belive, with this new package update `bytes` will also be part of the `progressEvent` right?

Reply:
Yes, you're right. https://github.com/axios/axios?tab=readme-ov-file#-progress-capturing